.github/workflows/lint.yaml

```diff
@@ -0,0 +1,30 @@
+name: "Lint"
```

Comment:
Do we just lint or eventually run more "code quality" checks?

Reply:
I think it's only linting so far, but we could aggregate those under a common flag eventually :-)